packages/suite/src/config/suite/yup.ts

```diff
@@ -0,0 +1,22 @@
+import { isHex as isHexUtil } from '@suite/utils/wallet/ethUtils';
+import { isAscii as isAsciiUtil } from '@trezor/utils';
+import * as yup from 'yup';
+
+yup.setLocale({
+    string: {
+        max: 'TR_EXCEEDS_MAX',
+    },
+    mixed: {
+        required: 'TR_REQUIRED_FIELD',
+    },
+});
+
+yup.addMethod<yup.StringSchema>(yup.string, 'isAscii', function isAscii() {
+    return this.test('isAscii', 'TR_ASCII_ONLY', value => isAsciiUtil(value));
+});
+
+yup.addMethod<yup.StringSchema>(yup.string, 'isHex', function isHex() {
+    return this.test('isHex', 'DATA_NOT_VALID_HEX', value => isHexUtil(value as string));
+});
+
+export default yup;
```

Comment:
The idea is to replace the usual yup import `import yup from 'yup'` with this, so to preserve the pattern I am using a default import. Anyways, I would also create an alias for it because the relative import from the config is ugly.

Reply:
Lot of libraries using default export but that doesn't mean that it is good. We should use named export, it will be easier to work with and since there is also exported Shape type now it will be better to do it unified `import { yup, SchemaShape }` than `import yup, { SchemaShape }`.